src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -47,11 +47,10 @@ object EntityClient {
 
     private def updatedEntities(
         url: Option[String],
-        token: String,
-        allEntities: Seq[Entity]
+        token: String
     ): F[Seq[Entity]] = {
       if (url.isEmpty) {
-        me.pure(allEntities)
+        me.pure(Nil)
```

Comment:
Do we even need this any more? Can you change `url` to be a String and get rid of the if statement. You're passing in the url directly so it should never be empty.